2018-05-17 17:24:24,080 ERROR: Exception on /reset_password_request [POST] [in /Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py:1753]
Traceback (most recent call last):
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 2284, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1807, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1710, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1805, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1791, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/app/routes.py", line 150, in reset_password_request
    send_password_reset_email(user)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/app/mail.py", line 12, in send_password_reset_email
    send_email('Microblog Password Reset', sender=app.config['ADMINS'], recipients=[user.email],
NameError: name 'app' is not defined
2018-05-17 17:24:58,459 INFO: Microblog startup [in /Users/KLM/Desktop/Programming/Webdev/microblog/app/__init__.py:40]
2018-05-17 17:25:11,969 ERROR: Exception on /reset_password_request [POST] [in /Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py:1753]
Traceback (most recent call last):
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 2284, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1807, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1710, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1805, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1791, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/app/routes.py", line 150, in reset_password_request
    send_password_reset_email(user)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/app/mail.py", line 13, in send_password_reset_email
    text_body=render_template('email/reset_password.txt', user=user, token=token),
NameError: name 'render_template' is not defined
2018-05-17 17:26:36,702 INFO: Microblog startup [in /Users/KLM/Desktop/Programming/Webdev/microblog/app/__init__.py:40]
2018-05-17 17:27:00,339 INFO: Microblog startup [in /Users/KLM/Desktop/Programming/Webdev/microblog/app/__init__.py:40]
2018-05-17 17:27:43,879 ERROR: Exception on /reset_password_request [POST] [in /Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py:1753]
Traceback (most recent call last):
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 2284, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1807, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1710, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1805, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1791, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/app/routes.py", line 150, in reset_password_request
    send_password_reset_email(user)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/app/mail.py", line 16, in send_password_reset_email
    html_body=render_template('email/reset_password.html', user=user, token=token))
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/app/mail.py", line 10, in send_email
    mail.send(msg)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask_mail.py", line 492, in send
    message.send(connection)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask_mail.py", line 427, in send
    connection.send(self)
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask_mail.py", line 188, in send
    self.host.sendmail(sanitize_address(envelope_from or message.sender),
  File "/Users/KLM/Desktop/Programming/Webdev/microblog/venv/lib/python3.6/site-packages/flask_mail.py", line 105, in sanitize_address
    nm, addr = addr
ValueError: not enough values to unpack (expected 2, got 1)
2018-05-17 17:30:58,625 INFO: Microblog startup [in /Users/KLM/Desktop/Programming/Webdev/microblog/app/__init__.py:40]
2018-05-17 17:37:23,668 INFO: Microblog startup [in /Users/KLM/Desktop/Programming/Webdev/microblog/app/__init__.py:40]
2018-05-17 17:38:54,855 INFO: Microblog startup [in /Users/KLM/Desktop/Programming/Webdev/microblog/app/__init__.py:40]
2018-05-17 17:39:05,813 INFO: Microblog startup [in /Users/KLM/Desktop/Programming/Webdev/microblog/app/__init__.py:40]
2018-05-18 16:14:23,744 INFO: Microblog startup [in /Users/KLM/Desktop/Programming/Webdev/microblog/app/__init__.py:40]
2018-05-18 16:19:24,665 INFO: Microblog startup [in /Users/KLM/Desktop/Programming/Webdev/microblog/app/__init__.py:40]
2018-05-18 16:42:43,779 INFO: Microblog startup [in /Users/KLM/Desktop/Programming/Webdev/microblog/app/__init__.py:40]
2018-05-18 16:46:40,085 INFO: Microblog startup [in /Users/KLM/Desktop/Programming/Webdev/microblog/app/__init__.py:40]
2018-05-18 16:50:02,073 INFO: Microblog startup [in /Users/KLM/Desktop/Programming/Webdev/microblog/app/__init__.py:40]
2018-05-19 12:57:50,795 INFO: Microblog startup [in /Users/KLM/Desktop/Programming/Webdev/microblog/app/__init__.py:42]
